executor/executor.go

```diff
@@ -769,6 +772,17 @@ func (e *LimitExec) Close() error {
 	return errors.Trace(e.baseExecutor.Close())
 }
 
+func (e *LimitExec) adjustRequiredRows(chk *chunk.Chunk) *chunk.Chunk {
```

Comment:
please add some comments about this function to explain how the number of child required rows is determined.

Reply:
more comments have been added.